### DISCUSSION TITLE
This comma may exist elsewhere

### DISCUSSION THREAD

**Discussion by higherkinded** [edenhill/librdkafka#3872] [General]
https://github.com/edenhill/librdkafka/blob/b47da0e4eec900d9cb091d2f524e957505dbb513/src-cpp/rdkafkacpp.h#L3209

Trailing commas are totally valid in C++ `enum` declarations. _This comma_ must exist between those members but it may be a trailing one after `RK_MSG_BLOCK` without any issues.

**Comment by edenhill** [comment]
If I remember correctly this was due to some old compiler complaining on trailing commas, perhaps an earler version of msvc.
